TSPL.docc/ReferenceManual/Expressions.md

```diff
@@ -1521,13 +1521,15 @@ A macro-expansion expression omits the parentheses after the macro's name
 if the macro doesn't take any arguments.
 
 A macro-expansion expression can't appear as the default value for a parameter,
-except the [`file()`][] and [`line()`][] macros from the Swift standard library.
-When used as the default value of a function or method parameter,
-these macros are evaluated using the source code location of the call site,
-not the location where they appear in a function definition.
+except the [`file()`][],  [`line()`][] and [`isolation()`][] macros from the Swift standard library.
```

Comment:
Typically, we write these macros without the parentheses, e.g., we'd call it the "file" macro or `#file`. Note that the restriction discussed here was actually lifted by [SE-0422](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0422-caller-side-default-argument-macro-expression.md), and any macro can appear as the default value for the parameter!

Reply:
Thanks, Doug — the spelling here matches how DocC renders this in API reference.  Compare <https://developer.apple.com/documentation/swift/isolation()> — maybe we should start a discussion with DocC folks about how macro names are spelled.

I'll rework this PR to lift the restriction, rather than expanding the list of special cases.

Reply:
Macros are a *little* weird in that one can omit the parentheses when expanding them, so long as it's permissible to provide zero arguments, so they're kinda between properties and functions.